### PR TITLE
feat: add Dropdown

### PR DESCRIPTION
**Pull request by Sekhmet**
Closes: https://github.com/snapshot-labs/sx-ui/issues/674
Depends on: https://github.com/snapshot-labs/sx-ui/pull/666

## Test plan
- Disable linea.
- Create proposal.
- You can open dropdown.

### CODE REVIEW

File: src/views/Proposal.vue [hunk 1]
```diff
@@ -185,24 +185,39 @@ watch(
...
-              </a>
-            </UiTooltip>
-            <UiTooltip title="View proposal metadata">
-              <a v-if="proposalMetadataUrl" :href="proposalMetadataUrl" target="_blank">
-                <IH-dots-vertical />
+          <UiDropdown class="mr-3">
+            <UiDropdownItem v-if="editable" v-slot="{ active }">
+              <button
+                class="flex items-center gap-2"
+                :class="{ 'opacity-80': active }"
```
Comment (bonustrack) [inline src/views/Proposal.vue:192]: Can we set this style within the Component itself so we don't need to replicate it elsewhere and we got same style across each dropdown?

Comment (bonustrack) [reply src/views/Proposal.vue:192]: When "active" is being used? Isn't this similar than "hover"? We can keep it like this if it's to complicated.

Comment (Sekhmet) [reply src/views/Proposal.vue:192]: Headless UI supports accessibility so active will be triggered if you focus in with keyboard, if you open popup and start typing part of word from item etc.

**Comment by bonustrack** [comment]
Can you add hand cursor on the 3 dot menu?

**Review by bonustrack — APPROVED** [review]
tACK

**Pull request merged**